extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -348,6 +358,36 @@ export class DatabaseUI extends DisposableObject {
     }
   };
 
+  private handleSetDefaultTourDatabase = async (
+    progress: ProgressCallback,
+    token: CancellationToken,
+  ): Promise<void> => {
+    try {
+      if (!workspace.workspaceFolders?.length) {
+        throw new Error("No workspace folder is open.");
+      } else {
+        // This specifically refers to the database folder in
+        // https://github.com/github/codespaces-codeql
+        const uri = Uri.parse(
+          `${workspace.workspaceFolders[0].uri}/codeql-tutorial-database`,
+        );
+
+        let databaseItem = this.databaseManager.findDatabaseItem(uri);
+        if (databaseItem === undefined) {
+          databaseItem = await this.databaseManager.openDatabase(
+            progress,
+            token,
+            uri,
+          );
+        }
+        await this.databaseManager.setCurrentDatabaseItem(databaseItem);
+      }
+    } catch (e) {
+      // rethrow and let this be handled by default error handling.
+      throw new Error(`Could not set database: ${getErrorMessage(e)}`);
```

Comment:
Could the error message be more precise and helpful? Something like this perhaps:

```suggestion
      throw new Error(`Could not set the database for the Code Tour. Please make sure you are default workspace of the CodeQL starter workspace in codespaces: ${getErrorMessage(e)}`);
```

Or something like that. Maybe even include a link. Most likely, if someone is seeing this error, they're not actually in the codespace, or maybe the code space didn't open the right workspace.